_config.yml

```diff
@@ -170,31 +181,6 @@ msapplication_tileimage:                'msapplication_tileimage.png'
 msapplication_tilecolor:                '#fabb00'
 
 
-#       ______                __                __
-#      / ____/___ _________  / /_  ____  ____  / /__
-#     / /_  / __ `/ ___/ _ \/ __ \/ __ \/ __ \/ //_/
-#    / __/ / /_/ / /__/  __/ /_/ / /_/ / /_/ / ,<
-#   /_/    \__,_/\___/\___/_.___/\____/\____/_/|_|      Facebook-Optimization › http://j.mp/fb_optimize
-
-og_image:                           'http://images.phlow.de/phlow/phlow_de-logo-512x.png'
-og_locale:                          'en_EN'
-og_type:                            'website'
```

Comment:
This stuff wasn't being used save for `og_locale`, which I missed with the SEO tag integration, which we can drive off the site locale instead. I can add the functionality back and open an issue against `jekyll-seo-tag`. Please let me know your thoughts.